framework/tester.py

```diff
@@ -166,11 +166,15 @@ def setUp(self):
         time.sleep(0.2)
 
     def tearDown(self):
+        tf_cfg.dbg(2, "\tTeardown")
```

Comment:
Debug messages for start up routines has the third level of debug. See the `setUp()` function right above. Probably `tearDown()` should use the same debug level.